server/routes/detail.js

```diff
@@ -0,0 +1,43 @@
+const express = require('express');
+const router = express.Router();
+const Detail = require('../../db/Detail.js');
+
+//get detail info for all houses
+router.get('/homes/all', (req, res) => {
+  Detail.find({})
+    .then(data => res.send(data))
+    .catch(error => res.sendStatus(500));
+});
+
+//get detail info for one house
+router.get('/homes/:home', (req, res) => {
+  Detail.findOne({ _index: req.params.home })
+    .then(data => res.send(data))
+    .catch(error => res.sendStatus(500));
+}); 
+
+//post detail info for one house
+router.post('/homes/:home', (req, res) => {
+  let body = req.body.data;
+  Detail.insert({ body })
+    .then(result => res.send(result))
```

Comment:
Also a nitpick, but you switched from using data in the past 2 methods to result here.

Reply:
The linter was mad at me for using data twice.